pom.xml

```
@@ -38,33 +35,40 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>junit</artifactId>
-      <version>1159.v0b_396e1e07dd</version>
     </dependency>
 
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>credentials</artifactId>
-      <version>2.6.1.1</version>
     </dependency>
 
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>plain-credentials</artifactId>
-      <version>1.5
-      </version>
     </dependency>
 
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>structs</artifactId>
-      <version>1.7</version>
     </dependency>
   </dependencies>
 
+  <dependencyManagement>
+      <dependencies>
+          <dependency>
+              <groupId>io.jenkins.tools.bom</groupId>
+              <artifactId>bom-2.361.x</artifactId>
+              <version>1981.v17df70e84a_a_1</version>

```

Optional would be nice to use the most recent release of the plugin bill of materials.
```suggestion
              <version>2025.v816d28f1e04f</version>
```